x-pack/test/rule_registry/common/lib/helpers/cleanup_target_indices.ts

```diff
@@ -17,7 +17,7 @@ export const cleanupTargetIndices = async (getService: GetService, user: User, s
     const aliasMap = await es.indices.getAlias({
       name: targetIndices,
       allow_no_indices: true,
-      expand_wildcards: 'open',
```

Comment:
hmmm ... I'm not sure `all` is right though either.  I wonder if `open` was used to prevent some of the RR init problems with non-hot-tiered indices (eg, `partial-*`), and so using `all` would make those scenarios more likely ...